torch_geometric/nn/conv/message_passing.py

```diff
@@ -85,22 +87,25 @@ class MessagePassing(torch.nn.Module):
         'size_i', 'size_j', 'ptr', 'index', 'dim_size'
     }
 
-    def __init__(self, aggr: Optional[Union[str, List[str]]] = "add",
-                 flow: str = "source_to_target", node_dim: int = -2,
-                 decomposed_layers: int = 1):
+    def __init__(
+        self,
+        aggr: Optional[Union[str, List[str], Aggregation]] = "add",
+        flow: str = "source_to_target",
+        node_dim: int = -2,
+        decomposed_layers: int = 1,
+    ):
 
         super().__init__()
 
-        if aggr is None or isinstance(aggr, str):
-            assert aggr is None or aggr in AGGRS
-            self.aggr: Optional[str] = aggr
-            self.aggrs: List[str] = []
+        if aggr is None or isinstance(aggr, (str, Aggregation)):
```

Comment:
TODO: Make SPMM aggrs such as 'sum' and SumAggregation behave the same

Reply:
Curious what the gap is today?

Reply:
The gap is that `self.aggr = str(SumAggregation())` is not the same as 'self.aggr = sum'. Therefore, `self.fuse` is `False` even through `SumAggregation()` should be able to be fused. 
https://github.com/pyg-team/pytorch_geometric/blob/0f7e018aff9f9ca5356bf90b27e3546e26343c37/torch_geometric/nn/conv/message_passing.py#L158

Ways to fix: 
1. Modify the `__repr__` of the fused `Aggregation` functions to return str like `sum`, `mean`, `max`, ...
2. Add their `__repr__` str to `FUSE_AGGRS`. 

Both if them seem to be tricky. Any suggestion?

Reply:
Yes, it is tricky. I'm not sure we desperately need to fix this right now, but one option would be to maintain a `repr` variable inside each `Aggregation` module for the reverse mapping. One could even automate that potentially (like a reverse `aggregation_resolver`)

Reply:
Added this to the roadmap.